exec/java-exec/src/main/java/org/apache/drill/exec/physical/base/AbstractBase.java

```diff
@@ -119,6 +119,16 @@ public void setMaxAllocation(long maxAllocation) {
     /*throw new DrillRuntimeException("Unsupported method: setMaxAllocation()");*/
   }
 
+  @Override
```

Comment:
It the size is a system option, then it need not be passed in each operator definition. Note that all operators should use the same batch size, so we'd never set this per operator. (The output of one operator is the input to another, and the inputs want to be controlled, we can only do that by controlling outputs.)